### ISSUE TITLE
PointRCNN config

### ISSUE THREAD

**Issue by brudermueller** [open-mmlab/OpenPCDet#381]
Dear @sshaoshuai 
I would love to have your advice on the parameter tuning in the pointrcnn config. I am currently training on the [JRDB Dataset ](https://jrdb.stanford.edu/dataset/about) for the human/pedestrian class. I have the feeling that the parameters in the target_config for the proposal_target_layer play a huge role (especially BG_THRESH, FG_THRESH in the classification and regression branch). How did you tune these parameters? 

Also looking at your uploaded pretrained models, what is the main difference between the _PointRCNN_ and the _PointRCNN-IoU_ model? Not many parameters have been changed in the configs but the performance on the pedestrian class of the PointRCNN-IoU model is significantly better according to your result table. 

Also, I have been trying to tune the learning rate parameters for the one-cycle decay. I have been reducing the div_factor to decrease the max_lr, do you think this makes sense? 

I would highly appreciate your input/thoughts. Thank you in advance.

**Comment by sshaoshuai** [comment]
It is hard to say how to tune these parameters, which depends on the data and specific problem. 

PointRCNN-IoU just uses the IoU metric for predicting the confidence of each box. You could refer to the codes for more details. 

In my experiments slightly modifying the learning rate doesn't significantly affect the final results.
